learning_resources/management/commands/mixins.py

```diff
@@ -0,0 +1,17 @@
+from learning_resources.models import LearningResource
+
+
+class TestResourceIdMixin:
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--test-ids",
+            dest="test_ids",
+            help="List of readable IDs to use for testing",
+        )
+
+    def configure_test_resources(self, options):
+        if options.get("test_ids"):
+            test_ids = options["test_ids"].split(",")
+            LearningResource.objects.filter(id__in=test_ids).update(
+                test_mode=True, published=False
```

Comment:
During the standard scheduled ETL pipeline runs, might be good to set `test_mode=False` if `published` gets set to `True` (indicating that the course is no longer in "test" mode and should no longer be treated differently than other courses).

Reply:
I've made it so that all the etl management commands (both file and data) do this at the end 
```python
LearningResource.objects.filter(published=True, test_mode=True).update(test_mode=False)
```